src/components/FilterSearch.tsx

```diff
@@ -123,14 +128,33 @@ export function FilterSearch({
   );
 
   useEffect(() => {
+    if (matchingFilters.length > 1 && !onSelect) {
+      console.warn('More than one selected static filter found that matches the filter search fields.'
+        + ' Please update the state to remove the extra filters.');
```

Comment:
what does the concatenation do vs a single string?

Reply:
This is just to split up the line so it doesn't become too long and exceed the max line length (linting error)